wandb/sdk/launch/sweeps/scheduler.py

```diff
@@ -123,22 +124,31 @@ def is_alive(self) -> bool:
             return False
         return True
 
-    def _try_load_job(self) -> bool:
-        _public_api = public.Api()
-        try:
-            _job_artifact = _public_api.artifact(self._sweep_config["job"], type="job")
-            wandb.termlog(
-                f"{LOG_PREFIX}Successfully loaded job: {_job_artifact.name} in scheduler"
+    def _try_load_executable(self) -> bool:
+        if self._kwargs.get("job"):
+            _public_api = public.Api()
+            try:
+                _job_artifact = _public_api.artifact(self._kwargs["job"], type="job")
+                wandb.termlog(
+                    f"{LOG_PREFIX}Successfully loaded job: {_job_artifact.name} in scheduler"
+                )
+            except Exception as e:
+                wandb.termerror(f"{LOG_PREFIX}{str(e)}")
+                return False
+            return True
+        elif self._kwargs.get("docker_image_uri"):
+            # TODO(gst): check docker existance? Maybe confirm NOT wandb run uri?
+            return True
+        else:
+            wandb.termerror(
```

Comment:
nit: I think this error should be outside of this function, then we throw the error when the function returns False. That way its more flexible in the future

Reply:
yup